# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1234212691 -3600
#      Mon Feb 09 21:51:31 2009 +0100
# Node ID 739141cde75a22b9ac9b5b327ea30c61fd357748
# Parent  d090213c697ab26db9c212525a7e9dbe2b848d0e
fix typo in Array-f.cc

diff --git a/liboctave/Array-f.cc b/liboctave/Array-f.cc
--- a/liboctave/Array-f.cc
+++ b/liboctave/Array-f.cc
@@ -37,17 +37,17 @@ along with Octave; see the file COPYING.
 #include "oct-sort.cc"
 
 template <>
 inline bool _sort_isnan (float x)
 {
   return lo_ieee_isnan (x);
 }
 
-INSTANTIATE_ARRAY_SORT (double);
+INSTANTIATE_ARRAY_SORT (float);
 
 INSTANTIATE_ARRAY_AND_ASSIGN (float, OCTAVE_API);
 
 INSTANTIATE_ARRAY_ASSIGN (float, int, OCTAVE_API)
 INSTANTIATE_ARRAY_ASSIGN (float, short, OCTAVE_API)
 INSTANTIATE_ARRAY_ASSIGN (float, char, OCTAVE_API)
 
 #include "Array2.h"
